app/src/main/java/ingsw/group1/findmyphone/MainActivity.java

```diff
@@ -40,6 +42,7 @@
     private Button sendButton;
     private Button sendAlarmRequestButton;
     private Button sendLocationRequestButton;
+    private Button viewContacts;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'viewContacts' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)